Makefile

```
@@ -119,13 +119,12 @@ vulncheck: $(BUILDDIR)/
 build-linux: go.sum
 	LEDGER_ENABLED=false GOOS=linux GOARCH=amd64 $(MAKE) build
 
-go-mod-cache: go.sum
-	@echo "--> Download go modules to local cache"
-	@go mod download
-
 go.sum: go.mod
 	@echo "--> Ensure dependencies have not been modified"
-	@go mod verify
+	go mod verify
```

P.S. `go mod verify` can be rather expensive; in our local clone I'm proposing to make this step suppressable.
```suggestion
	@echo "--> Ensure dependencies have not been modified unless suppressed by SKIP_MOD_VERIFY"
ifndef SKIP_MOD_VERIFY
	go mod verify
endif
```